forwards_paper.tex

```diff
@@ -656,6 +661,27 @@ \subsection*{Recording the pedigree in forwards time}
 ancestors in the current population, and so the entire path tracing back to
 its common ancestor with 27 is redundant.
 
+\paragraph{Essential steps in tree sequence recording}
+To record a tree sequence, including mutations,
+a simulator must record:
+\begin{itemize}
+    \item the birth time of each individual in the Node Table,
+    \item the genetic segments each individual inherits from her parents in the Edge Table, and
+    \item the individual each new mutation occurs in, and the derived state in the Mutation Table,
+        adding also to the Site Table if the site is previously unmutated.
+\end{itemize}
+Recall that here we mean ``\emph{haploid} individual'', i.e., gamete.
+Before simplification, time must be translated to ``time ago'' if it is not already.
+which was avoided in Algorithm~\algref{W} since there $t$ denoted
+``time until the end of the simulation''.
+Furthermore, since $\tsimplify{P', \Nt, \Et}$ results in the $N$ individuals in $P'$
+being relabeled in \msprime{}'s tables as $0, 1, \ldots, N-1$,
+there must be bookkeeping that keeps in sync the individual IDs as recorded by the simulator
+with the node IDs recorded in the tables.
+% Finally, the first step in simplification is to \emph{sort} the tables
```

Comment:
I don't think we need to explain here about sorting, actually: [the documentation](https://github.com/jeromekelleher/msprime/blob/master/msprime/tables.py#L1243) explains this, and it's not an "essential" step in that other implementations of the same ideas would conceivably not need to do this.